docs/docs/tutorials/rill_advanced_features/custom_dashboards/getting-started.md

```diff
@@ -0,0 +1,59 @@
+---
+title: "Custom Dashboards"
+description: Creating Custom Dashboards in Rill
+sidebar_label: "Getting Started"
+sidebar_position: 6
+---
+
+## Creating Custom Dashboards in Rill 
+
+In this section, we'll cover how to create custom dashboards in Rill Developer and publish these to Rill Cloud,
+
+:::note
```

Comment:
Should go live in 0.50 so we could probably omit this section if we intend the tutorials to go out at the same time